torch/fx/traceback.py

```diff
@@ -0,0 +1,54 @@
+from contextlib import contextmanager
+from typing import Optional, List
+from ._compatibility import compatibility
+
+__all__ = ['override_stack_trace', 'append_stack_trace', 'format_stack', 'is_stack_trace_overridden']
+
+current_stack: List[str] = []
```

Comment:
there seems to be some stack confusion here: is this a list of lines of a stacktrace, or a list of entire stacktraces. Looks like the former, from reading the code

Reply:
this is stack of stacks, the latter one.

Reply:
This doesn't seem consistent with the `append_stack_trace` then, which seems to push an entire stack trace (all lines)

Reply:
OK, I was confused by "list of lines of a stacktrace" and "list of entire stacktraces"... 

the content of stack variable here is something like this 

```
  File "/fsx/users/bahuang/repos/pytorch_fsx/test_traceback.py", line 27, in <module>
    f(1)
  File "/fsx/users/bahuang/repos/pytorch_fsx/test_traceback.py", line 6, in f
    return ff(a)
  File "/fsx/users/bahuang/repos/pytorch_fsx/test_traceback.py", line 9, in ff
    return fff(a)
  File "/fsx/users/bahuang/repos/pytorch_fsx/test_traceback.py", line 12, in fff
    stack = traceback.print_stack()
```

I suppose this the "entire stack trace (all lines)"